docs/standard/serialization/json-serialization-how-to.md

```diff
@@ -0,0 +1,755 @@
+---
```

Comment:
I'd remove serialization from the file name since you already have serialization in the path, to avoid duplication (SEO)